### ISSUE TITLE
Windows XP - owncloud.exe error

### ISSUE THREAD

**Issue by lippoliv** [owncloud/client#2941]
### Expected behaviour
ownCloud should start...

### Actual behaviour
ownCloud crashes on System-Boot (cauz it should auto-start) and on Manual start.

### Steps to reproduce
1. use WIN XP
2. install ownCloud 1.7.1
3. Start it

### Server configuration
Operating system: Debian 7

Web server: Apache

Database: MySQL

PHP version: 5.4.36

ownCloud version: 7.0.4

Storage backend: ?

### Client configuration
Client version: 1.7.1

Operating system: Windows XP

OS language: DE

Installation path of client: C:\Programme\ownCloud (Standard)

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

```Template for output < 10 lines```

1. Output of `owncloud --logwindow` or `owncloud --logfile log.txt`
(On Windows using `cmd.exe`, you might need to first `cd` into the ownCloud directory)
(See also http://doc.owncloud.org/desktop/1.5/troubleshooting.html#client-logfile )
NOTHING...

![unbenannt](https://cloud.githubusercontent.com/assets/2561859/6585009/75a455b4-c76f-11e4-9e3d-60bb7d26b083.PNG)

2. Web server error log: nothing

3. ownCloud log (data/owncloud.log): 
No such file!?

**Comment by dragotin** [comment]
Please retry with ownCloud 1.8.0 beta2 if you still see this. Thx.

**Comment by lippoliv** [comment]
@dragotin do you have an download-link?

**Comment by ckamm** [comment]
@lippoliv The testing packages are listed at the bottom of https://owncloud.org/install/ (Windows: https://download.owncloud.com/desktop/testing/ownCloud-1.8.0.4806beta2-setup.exe).

**Comment by danimo** [comment]
https://owncloud.org/install/#testing should get your there.

**Comment by JosefKrieglstein** [comment]
Hi, I can confirm same failed to start bug with same 0xc000001d message. 1.6.4, 1.7.0, 1.7.1 all these versions affected. My computer: Windows XP SP3, 32bit, Czech. Last working version: 1.6.3 build:3721.
Client 1.8.0.4806beta2 did not start too, with same message 0xc000001d.

Josef

**Comment by lippoliv** [comment]
Also in my Case that beta doesn't start up...

Is there an Archive available where I can download 1.6.1 or older???

**Comment by JosefKrieglstein** [comment]
https://download.owncloud.com/desktop/stable/ownCloud-1.6.3.3721-setup.exe
Running Owncloud want to download and install new version, you must stop downloading (by killing the proces) new version when it starts or manualy modify C:\Documents and Settings\%USERNAME%\Local Settings\Data aplikací\ownCloud\owncloud.cfg

My 3 last lines of owncloud.cfg:
[Updater]
autoUpdateFailedVersion=1.7.1.4382
seenVersion=1.7.1.4382

Josef

**Comment by danimo** [comment]
Hi, let's please try to sort this out. We definitely want to support XP (SP3), and last time I have tried (before 1.8.0 rc1), it worked for me. Please try RC1 from https://owncloud.org/install/#testing.

**Comment by JosefKrieglstein** [comment]
On https://owncloud.org/install/#testing, there is only 1.8.0.4806beta2. This version is also affected.
Josef

**Comment by danimo** [comment]
@JosefKrieglstein My apologies, rc1 is now linked.

**Comment by JosefKrieglstein** [comment]
It is still same. The crash happen after all *.dll library in same folder is checked, except libssl-10.dll, Owncloud.exe did not check existence of libssl-10.dll. Can I do some kind of debug?
Josef

**Comment by danimo** [comment]
> The crash happen after all *.dll library in same folder is checked

What do you mean by that? Can you describe exactly what you are doing? I just tested with a plain XP SP3 and it works just fine.

**Comment by JosefKrieglstein** [comment]
I think I have also plain XP, but CZECH language version, and all updates (www.windowsupdates.com).
I try to delete file by file from C:\program files\Owncloud\, the crash occurs with twice popup message when there are owncloud.exe and all dll except libssl-10.dll, this library may not be in this folder. When I delete another dll, only one popup message is showed: missing ...dll. I only trying help you at what point crash occur. When owncloudcmd.exe -v is run from command line, no output in cmd is displayed. Only crash twice popup message.

**Comment by fabeblau** [comment]
Hi, same here with the rc1-clinet, see image. SP3 is installed, but it's a _very old_ machine I'm still using here..
![clipboard01](https://cloud.githubusercontent.com/assets/3115204/6656793/0f320978-cb36-11e4-85c6-ddc97f986b9b.jpg)

**Comment by danimo** [comment]
@fabeblau @JosefKrieglstein This seems to be related to a lack of CPU instruction sets. Which exact CPU model are you using?

**Comment by JosefKrieglstein** [comment]
AMD Duron:
processor       : 0
vendor_id       : AuthenticAMD
cpu family      : 6
model           : 7
model name      : AMD Duron(tm) processor
stepping        : 1
cpu MHz         : 1298.000
cache size      : 64 KB
fpu             : yes
fpu_exception   : yes
cpuid level     : 1
wp              : yes
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 mmx fxsr sse syscall mmxext 3dnowext 3dnow
TLB size        : 512 4K pages
clflush size    : 64
cache_alignment : 64
address sizes   : 34 bits physical, 32 bits virtual
power management: ts

is it enough?

**Comment by danimo** [comment]
It seems this CPU does not provide SSE2 and ownCloud Client expects SSE2 to be present. This is most unrelated to libssl. Not having SSE2 means your hardware is > 10 years old, and neglecting the benefits of at least moderately modern instruction sets will significantly slow down performance. Also l ibraries that ownCloud client depends on have chosen to depend on SSE2 by default.

Unfortunately, there is no good way to signal this.

**Comment by JosefKrieglstein** [comment]
Thank you very much that you found it. You right this CPU has not SSE2.

**Comment by fabeblau** [comment]
No SSE2 here either, whatever that is. Thanks a lot @danimo , perhaps it's really time to move on for me ;-)

**Comment by vader2014** [comment]
i have the same issue. Currently i'm running v.1.5.4 of the client on an old maschine used for backup purposes. Would be nice to have a build that does not relay on SSE2.

**Comment by danimo** [comment]
@vader2014 Nope, sorry.

**Comment by vader2014** [comment]
In the light of https://github.com/owncloud/core/pull/15683 this issue will become an issue. After some research I think the problem is related to QT5. As there is the option to enforce QT4 (BUILD_WITH_QT4=ON) in the build process, would it be possible to have such a build (currently my main systems run on Ubuntu, so I'm not sure if the build instructions would work in this environment)?

**Comment by danimo** [comment]
@vader2014 Ubuntu ships with a Qt 4 based client.

**Comment by vader2014** [comment]
Sorry I was not clear enough: I would like to try if a windows client with QT5 would run on my old windows box.

**Comment by danimo** [comment]
You mean Qt4? Qt4 for Windows and OS X is out of the question. We rely on Qt5 APIs there. Do you even run a supported Windows version on those machines? (i.e. >Windows Vista?)

**Comment by vader2014** [comment]
Yes QT4. No supported Windows, still on XP in a very specific productive environment (special network segment with no public network access, disabled USB storage drivers) :-(
OC was a very convenient near-real time way to sync data to this box.

**Comment by danimo** [comment]
Note that Qt5 can be compiled without SSE2 support, all I'm saying that I don't consider this a change I would like to see in our normal packages. However you can fork https://build.opensuse.org/package/show/windows:mingw:win32/mingw32-libqt5-qtbase and change the configure parameters in the spec file. The resulting libraries should then work (but may not, since some functions are inlined).

**Comment by vader2014** [comment]
THX, I will have a look there.

**Comment by rienesl** [comment]
My first install of Owncloud on a Windows machine and it turns out that it is impossible to run the latest version because of the lack of a cpu instructionset, developed for vector calculation...nice...
Gave version 1.6.3 a try (like mentioned above): installation and configuration is possible, but this version seems to have serious problems connecting to a 8.0.4 server (status remains 99% of the time in "disconnected from server", no sync ever happened). Currently I'm not quite sure if I should continue troubleshooting, because this version does not support selective sync!
